packages/deploy/src/client.ts

```diff
@@ -37,6 +42,30 @@ export async function getProject(
   }
 }
 
+export async function downloadSpec(
+  config: DeployConfig,
+  projectId: string
+): Promise<{ data: ProjectPayload | null }> {
+  let base = new URL(config.endpoint).origin; 
+  let url = new URL(base + `/download/yaml`);
+  url.searchParams.set('id', projectId);
+
+
+  try {
+      const res = await fetch(url);
+      const destination = path.resolve(".", config.specPath);
```

Comment:
You shouldn't need that leading `.`, as `resolve` will resolve from the current working directory anyway. I'd also condense these lines, like so:
```
const fileStream = fs.createWriteStream(path.resolve(config.specPath), {flags: 'wx'});
```